op-acceptance-tests/tests/base/chain_test.go

```diff
@@ -0,0 +1,48 @@
+package base
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/system"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/systest"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+// TestChainFork checks that the chain does not fork.
+func TestChainFork(t *testing.T) {
+	chainIdx := uint64(0)
```

Comment:
Why not loop over all chains in the devnet?

Bonus points if we do that in parallel.